src/pkg/reg/adapter/aliacr/adapter.go

```diff
@@ -295,31 +293,26 @@ func (a *adapter) FetchArtifacts(filters []*model.Filter) (resources []*model.Re
 				return fmt.Errorf("list tags for repo '%s' error: %v", repo.RepoName, err)
 			}
 
-			var filterTags []string
-			if tagsPattern != "" {
-				for _, tag := range tags {
-					var ok bool
-					ok, err = util.Match(tagsPattern, tag)
-					if err != nil {
-						return fmt.Errorf("match tag '%s' error: %v", tag, err)
-					}
-					if ok {
-						filterTags = append(filterTags, tag)
-					}
-				}
-			} else {
-				filterTags = tags
+			var artifacts []*model.Artifact
+			for _, tag := range tags {
+				artifacts = append(artifacts, &model.Artifact{
```

Comment:
this is a reference to other existing implementation like native adapter `https://github.com/goharbor/harbor/blob/eabff82366afdf464c593046f3b7a60434399880/src/pkg/reg/adapter/native/adapter.go#L222`
and I think it's because we just use the tag name, no other field like `digest`, so it's no problem to use 1:1 relationship